src/utils/test/render-markdown-test.js

```diff
@@ -1,21 +1,29 @@
+import sinon from 'sinon';
+
 import { renderMathAndMarkdown, $imports } from '../render-markdown';
 
 describe('render-markdown', () => {
   let render;
+  let fakeRenderToString;
+  let fakeEscapeHtml;
```

Comment:
`render-markdown.ts` is old code from before we standardized casing, it should be `escapeHTML` in the tests and implementation.